Popov_XO_test/Popov_XO_test/PlayField.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <vector>
+#include <initializer_list>
+
+class PlayField {
+public:
+    enum CellStatus { csEmpty, csCross, csNought };
+    enum FieldStatus {
+        fsInvalid,
+        fsNormal,
+        fsCrossesWin,
+        fsNoughtsWin,
+        fsDraw
+    };
+
+    class CellPos {
+    public:
+        CellPos(std::initializer_list<int> cell) {
+            _x = *cell.begin();
+            _y = *(cell.begin() + 1);
+        }
+        const int getX() { return _x; }
+        const int getY() { return _y; }

```

Comment:
Да блин! Надо вам ТЗ писать, чтобы не могли иначе, ёмаё.
```suggestion
        const int getX() const { return _x; }
        const int getY() const { return _y; }

```